src/main.py

```diff
@@ -0,0 +1,78 @@
+# sudoku_solver.py
+# author: Daniel Jauergui
+# date: 3-31-2015
+
+from configuration.configuration import *
+from file_manager.file_manager import *
+from menu import Menu
+from game import *
+from board import *
+
+
+def display_main_menu():
+    menu = Menu('Sudoku Solver')
+    menu.text = "Sudoku is a puzzle game designed for\na single player, much like a crossword puzzle."
+    menu.clear_items()
+    menu.add_item((1, 'Configuration Game', display_config_menu))
+    menu.add_item((2, 'Start Game', game_menu))
+    menu.add_item((0, 'Exit', None))
+    menu.ask()
+
+
+def display_config_menu():
+    menu = Menu('Sudoku Solver - Config Section')
+    menu.clear_items()
+    menu.add_item((1, 'Select Level', display_config_menu))
+    menu.add_item((2, 'Select Algorithm', display_config_menu))
+    menu.add_item((3, 'Print Config File', print_config_file))
+    menu.add_item((4, 'Print Config Level', print_level_file))
+    menu.add_item((5, 'Back', display_main_menu))
+    menu.add_item((0, 'Exit', None))
+    menu.ask()
+
+
+def print_config_file():
+    os.system('cls')
+    print ('Print Configuration file\n\n')
+    file_obj = File('configuration/xml_config.xml')
+    config = Configuration(file_obj.read_content())
+    print (config.get_xml_as_string())
+    raw_input('\n\nPress any key: ')
+    display_config_menu()
+
+
+def print_level_file():
+    os.system('cls')
+    print ('Print Level of Configuration file\n\n')
+    file_obj = File('configuration/xml_config.xml')
+    config = Configuration(file_obj.read_content())
+    print (config.level)
+    raw_input('\n\nPress any key: ')
+    display_config_menu()
+
+
+def game_menu():
```

Comment:
this method name is not a verb, remember that classes are nouns and methods are verbs